modules/ptp-dual-nics.adoc

```diff
@@ -6,38 +6,47 @@
 [id="ptp-dual-nics_{context}"]
 = Using dual-NIC Intel E810 hardware with PTP
 
-{product-title} supports single and dual-NIC Intel E810 hardware for precision PTP timing in grandmaster clocks (T-GM) and boundary clocks (T-BC).
+{product-title} supports single and dual-NIC Intel E810 hardware for PTP timing in grandmaster clocks (T-GM) and boundary clocks (T-BC).
 
 Dual NIC grandmaster clock::
++
+--
 You can use a cluster host that has dual-NIC hardware as PTP grandmaster clock.
 One NIC receives timing information from the global navigation satellite system (GNSS).
 The second NIC receives the timing information from the first using the SMA1 Tx/Rx connections on the E810 NIC faceplate.
 The system clock on the cluster host is synchronized from the NIC that is connected to the GNSS satellite.
-+
+
 Dual NIC grandmaster clocks are a feature of distributed RAN (D-RAN) configurations where the Remote Radio Unit (RRU) and Baseband Unit (BBU) are located at the same radio cell site.
 D-RAN distributes radio functions across multiple sites, with backhaul connections linking them to the core network.
-+
+
 .Dual NIC grandmaster clock
-image::561_OpenShift_Using_PTP_network_0124.png[Dual NIC PTP grandmaster clock connected to GNSS timing source and downstream PTP boundary and ordinary clocks]
-+
+image::openshift-ptp-using-dual-nic-ptp.png[Dual NIC PTP grandmaster clock connected to GNSS timing source and downstream PTP boundary and ordinary clocks]
+
 [NOTE]
 ====
-In a dual-NIC T-GM configuration, a single `ts2phc` process reports as two `ts2phc` instances in the system.
+In a dual-NIC T-GM configuration, a single `ts2phc` program operate on two PTP hardware clocks (PHCs), one for each NIC.
 ====
+--
 
 Dual NIC boundary clock::
-For 5G telco networks that deliver mid-band spectrum coverage, each virtual distributed unit (vDU) requires connections to 6 radio units (RUs). To make these connections, each vDU host requires 2 NICs configured as boundary clocks.
 +
+--
+For 5G telco networks that deliver mid-band spectrum coverage, each virtual distributed unit (vDU) requires connections to 6 radio units (RUs). To make these connections, each vDU host requires 2 NICs configured as boundary clocks.
+
 Dual NIC hardware allows you to connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding the downstream clocks.
+--
 
 Highly available system clock with dual-NIC boundary clocks::
++
+--
 You can configure Intel E810-XXVDA4 Salem channel dual-NIC hardware as dual PTP boundary clocks that provide timing for a highly available system clock.
 This is useful when you have multiple time sources on different NICs.
 High availability ensures that the node does not lose timing synchronisation if one of the two timing sources is lost or disconnected.
```

Comment:
```suggestion
High availability ensures that the node does not lose timing synchronization if one of the two timing sources is lost or disconnected.
```
Although I didn't find explicit advice stating that we need to use the American English "synchronization" instead of "synchronisation", IBM Style uses the American English spelling in word usage: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#sync. It also looks like we're using "synchronize" in other places this PR.